<?xml version="1.0"?>
<launch>

    <!-- Spawns a robot in the simulation and runs its robot_state_publisher and a global planner -->
    <arg name="robot" default="robot"/>
    <arg name="radius" default="0.2"/>
    
    <node pkg="map_simulator" type="spawn" name="spawn" output="screen">
        <param name="static_tf_odom" value="true"/>
        <param name="force_scanner" value="false"/>
        <param name="radius" value="$(arg radius)"/>
        <param name="x" value="$(arg x)"/>
        <param name="y" value="$(arg y)"/>
        <param name="theta" value="$(arg theta)"/>
    </node>
    
</launch>
